Core/Frameworks/Baikal/Model/Config/Database.php

```diff
@@ -82,6 +87,34 @@ function formMorphologyForThisModelInstance() {
             "label" => "MySQL password",
         ]));
 
+        $oMorpho->add(new \Formal\Element\Checkbox([
+            "prop" => "pgsql",
+            "label" => "Use PostgreSQL",
```

Comment:
Selecting the database should be a dropdown. Otherwise you can have both mysql and pgsql checked on the settings page.

Reply:
Done, there's a dropdown for the database backend now in 4ab75f76. This replaces the `mysql` and `pgsql` checkboxes and connected config values.